schema/samcli.json

```diff
@@ -143,9 +143,175 @@
               "type": "object",
-              "properties": {}
+              "properties": {
+                "config_env": {
+                  "title": "config_env",
+                  "type": "string",
+                  "description": "Environment name specifying default parameter values in the configuration file.",
+                  "default": "default"
+                },
+                "config_file": {
+                  "title": "config_file",
+                  "type": "string",
+                  "description": "Configuration file containing default parameter values.",
+                  "default": "samconfig.toml"
+                },
+                "guided": {
+                  "title": "guided",
+                  "type": "boolean",
+                  "description": "Specify this flag to allow SAM CLI to guide you through the deployment using guided prompts."
+                },
+                "template_file": {
+                  "title": "template_file",
+                  "type": "string",
+                  "description": "AWS SAM template which references built artifacts for resources in the template. (if applicable)",
+                  "default": "template.[yaml|yml|json]"
```

Comment:
Is `template.[yaml|yml|json]` really the default value? or a symbol representing that it will look up these three files in certain order?

Reply:
It is, yes. I thought it weird, but that's how the default value is stored in the command, I assume so it's more readable under the CLI as options.